### DISCUSSION TITLE
Bundler In CodeSandbox Takes Forever

### DISCUSSION THREAD

**Discussion by fweth** [josdejong/mathjs#2729] [Q&A] [answered]
Hello, I don't know if this is a problem of CSB or Math.js, but the bundler takes forever to bundle a few functions from Math.js, I never saw this with any other library before.

https://codesandbox.io/s/thirsty-jones-zs826h

Or maybe it's just my browser, but sometimes the CSB even freezes before finishing the bundling process.  Any ideas?

**Comment by josdejong** [comment, marked as answer]
I just tried, it works on my machine right now, and making a small change refreshes almost instantly. 

I have to say I regularly have stability issues with CodeSandbox, hard to say what the cause is.

**Reply by fweth** [reply]
Thanks a lot for the answer!  Yes, sometimes it refreshes instantly, but sometimes it takes 1 minute or so.  But I'd also say it's more of CodeSandbox's problem, it could at least cache the bundle or something, as long as I don't change anything on the imports, it shouldn't rerun the bundler imo.
